webapp/src/index.js

```diff
@@ -0,0 +1,18 @@
+import UserAttribute from './components/user_attribute';
+import Reducer from './reducers';
+import {id as pluginId} from './manifest';
+
+class Plugin {
+    async initialize(registry) {
+        registry.registerReducer(Reducer);
+
+        registry.registerPopoverUserAttributesComponent(UserAttribute);
+    }
+
+    deinitialize() {
+        //eslint-disable-next-line no-console
+        console.log('Shutting down cloud plugin');
```

Comment:
Debugging message?

Reply:
This is actually nearly identical to what the demo plugin does. I don't know how useful it is to keep or not.

Reply:
I think that's just in the demo plugin to show that there is a deinit function that can be used